skopt/optimizer/optimizer.py

```diff
@@ -333,13 +334,27 @@ def ask(self, n_points=None, strategy="cl_min"):
         for i in range(n_points):
             x = opt.ask()
             X.append(x)
+
+            ti = [t for (_, t) in opt.yi] if "ps" in self.acq_func else None
+
             if strategy == "cl_min":
                 y_lie = np.min(opt.yi) if opt.yi else 0.0  # CL-min lie
+                t_lie = np.min(ti) if ti else sys.float_info.max
```

Comment:
Another issue: `sys.float_info.max` should be `log(sys.float_info.max)`. If we did not use `_tell()` but `tell()`, the value would be log transformed. Will probably not make a difference in practice, but i feel this closer to the rest of the idea.

Reply:
Probably also less numerical issues with surrogate which estimates time